### ISSUE TITLE
Remove the rest of the `Browser` namespace

### ISSUE THREAD

**Issue by jonkoops** [Leaflet/Leaflet#8774]
### Checklist

- [X] I've searched through the [plugins](https://leafletjs.com/plugins.html) to make sure this feature isn't already available, or think it shouldn't require a plugin.
- [X] I've searched through the [current issues](https://github.com/Leaflet/Leaflet/issues) to make sure this feature hasn't been requested already.
- [X] I agree to follow the [Code of Conduct](https://github.com/Leaflet/Leaflet/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.

### Motivation

I've been able to remove most of the APIs in the [`Browser`](https://github.com/Leaflet/Leaflet/blob/0042d0b0ddac8e9159ee4f64742bb25b518b9e0f/src/core/Browser.js) namespace. Some of these APIs still remain due to some of our code still relying on platform and browser specific detection.

The following APIs in the `Browser` namespace are used to calculate the [scroll wheel to pixel factor](https://github.com/Leaflet/Leaflet/blob/0042d0b0ddac8e9159ee4f64742bb25b518b9e0f/src/dom/DomEvent.js#L266-L273):

- `linux`
- `chrome`
- `mac`

We need to find a way to do this calculation without resorting to detecting specific platforms.

### Suggested solution

Foo

**Comment by Falke-Design** [comment]
I don't think that there is a way around the Browser detection. The problem is that each Browser handles it different so we need to handle different for the browsers

**Comment by IvanSanchez** [comment]
This should be on hold until #8859 is resolved.
